test/README.md

```diff
@@ -0,0 +1,41 @@
+# Testing
+
+## End-to-end testing
+
+The test uses KinD with MetalLB, so external traffic can be routed into
+the cluster.
+
+### How to use it
+
+To run the tests locally, run the following commands:
+
+1. set up a test env (creates a kind cluster with MetalLB and pre-loads docker images):
+
+       make e2e-test-env
+
+2. run the tests (builds the istio-operator docker image, installs it using helm and starts the tests):
+
+       make TAG=dev e2e-test
+
+The "TAG" variable is used for the istio-operator docker image tag. It can be anything, but you should
```

Comment:
If it can cause mixups shouldn't we randomize it maybe? Where does this tag matter at all?

Reply:
I wrote a short story about this. Hopefully, it makes more sense than the above.